cmd/kubeadm/app/master/apiclient.go

```diff
@@ -35,22 +35,35 @@ import (
 
 const apiCallRetryInterval = 500 * time.Millisecond
 
-func CreateClientAndWaitForAPI(adminConfig *clientcmdapi.Config) (*clientset.Clientset, error) {
+func createAPIClient(adminKubeconfig *clientcmdapi.Config) (*clientset.Clientset, error) {
 	adminClientConfig, err := clientcmd.NewDefaultClientConfig(
-		*adminConfig,
+		*adminKubeconfig,
 		&clientcmd.ConfigOverrides{},
 	).ClientConfig()
 	if err != nil {
-		return nil, fmt.Errorf("<master/apiclient> failed to create API client configuration [%v]", err)
+		return nil, fmt.Errorf("<cmd/token> failed to create API client configuration [%v]", err)
```

Comment:
This isn't a token thing, necessarily.  This class will get used lots of places.

We should either pass the "phase" in or prepend the phase as we dump the error.